joomla/joomla-cms
# Finally com_mailto allow the usage of a captcha by using JForm

### Summary of Changes

Finally com_mailto allow the usage of a captach by using JForm 

As discussed with the JSST this is now a public tracker item: cc @SniperSister @mbabker

### Testing Instructions

- apply this patch
- check that the mail to friend feature still works
- configure the captcha
- check that com_mailto now requires the captcha

### Expected result

com_mailto supports captcha's
com_mailto uses JForm

### Actual result

com_mailto don't supports captcha's
com_mailto don't uses JForm

### Documentation Changes Required

none
